ci/src/main/scala/org/typelevel/sbt/CrossRootProject.scala

```diff
@@ -99,11 +99,13 @@ object CrossRootProject {
   def unapply(root: CrossRootProject): Some[(Project, Project, Project, Project)] =
     Some((root.all, root.jvm, root.js, root.native))
 
-  private[sbt] def apply(): CrossRootProject = new CrossRootProject(
-    Project("root", file(".")),
-    Project("rootJVM", file(".jvm")),
-    Project("rootJS", file(".js")),
-    Project("rootNative", file(".native"))
+  def apply(): CrossRootProject = CrossRootProject("root")
+
+  def apply(name: String): CrossRootProject = new CrossRootProject(
```

Comment:
Nit, it's technically the `id` :)

```suggestion
  def apply(id: String): CrossRootProject = new CrossRootProject(
```